### ISSUE TITLE
Rule proposal: `min-lines`

### ISSUE THREAD

**Issue by MatanGbr** [sindresorhus/eslint-plugin-unicorn#1696]
### Description

I know it sounds weirds, but I believe there is no place for really short JavaScript file, and that these tiny bits of code should be combined with other tiny bits of code to make a file of at least 5 lines.

### Fail

```js
function foo() {
  var replace = 'me';
  return replace;
}
```

### Pass

```js
function foo() {
  var replace = 'me';
  var x = 5;
  var y = 6;
  return replace + x;
}
```

**Comment by papb** [comment]
@MatanGbr it's nice that you tried to start coding this rule, but as @fisker already said, [this is not the place to ask for help on coding rules](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1695#issuecomment-1014564514). You should edit your post to remove most of the *Description* section, leaving only this:

> ### Description
> 
> I know it sounds weirds, but I believe there is no place for really short JavaScript file, and that these tiny bits of code should be combined with other tiny bits of code to make a file of at least 5 lines.

**Comment by papb** [comment]
That said, @fisker already gave an [example](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1695#issuecomment-1014405316) on why he disagrees with this rule. I also don't agree with this rule, I think super short modules are very valid. Sindre has many of those himself, so I doubt this rule will be accepted...

**Comment by fisker** [comment]
Commented out the rule implement parts, anyway here is how you write your rule, but please don't ask for similar help in future.


<details>
<summary>This should work</summary>

```js
'use strict'
module.exports = {
  meta: {
    type: 'suggestion',
    schema: [],
  },
  create(context) {
    const sourceCode = context.getSourceCode();
    return {
      "Program:exit"(node) {
        if (sourceCode.lines.length < 5) {
          context.report({
            node: node,
            message: 'File should have more lines',
          })
        }
      },
    }
  },
}
```
</details>

**Comment by MatanGbr** [comment]
Ok. thanks man!
